src/Services/SubscriptionService.php

```diff
@@ -125,4 +126,33 @@ public function switchTo($subscriptionId, $data)
             ['subscriptionId' => $subscriptionId]
         );
     }
+
+    /**
+     * @param string $subscriptionId
+     *
+     * @return LeadSource
+     */
+    public function getLeadSource($subscriptionId)
+    {
+        return $this->client()->get('subscriptions/{subscriptionId}/lead-source', ['subscriptionId' => $subscriptionId]);
+    }
+
+    /**
+     * @param string $subscriptionId
+     * @param LeadSource $leadSource
```

Comment:
This can be not only `LeadSource`, but array or `JsonSerializable` (as it made in ither services)